requirements/devel.txt

```diff
@@ -10,6 +10,6 @@ setuptools
 lxml
 # model graph
 # requires: graphviz-devel
-pygraphviz
+pygraphviz<1.3
```

Comment:
I would suggest having a single patch to fix it, but not include this change in every patch.